.openshiftio/Jenkinsfile.setup.groovy

```diff
@@ -0,0 +1,11 @@
+
+def setupEnvironmentPre(env) {
+  sh "oc policy -n ${env} add-role-to-user view -z default"
+  sh "if ! oc get -n ${env} configmap app-config -o yaml | grep application.properties; then oc create -n ${env} configmap app-config --from-file=src/main/etc/application.properties; fi"
```

Comment:
Shouldn't the path be `greeting-service/src/main/etc/application.properties`?

Reply:
Ok, updated